tools/install_rust_msrv.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if [ ! -d rust-installer ]; then
+    mkdir rust-installer
+    wget https://sh.rustup.rs -O rust-installer/rustup.sh
+    sh rust-installer/rustup.sh -y -c llvm-tools
+    rustup default 1.79
+    rustup component add llvm-tools
+fi
+. "$HOME/.cargo/env"
```

Comment:
Since this is set to the MSRV, we probably also should include a link to it in `Cargo.toml`'s `rust-version` field, where we keep track of all the places that need updating.  I could easily see this getting forgotten on an MSRV bump, and I don't think we habitually build the wheels on those commits.

Reply:
Added in: https://github.com/Qiskit/qiskit/pull/13737/commits/20f7e0a820678b21efc5f081adab96e42f943add